src/provider.rs

```diff
@@ -73,11 +73,25 @@ pub fn load_image_from_path<P: AsRef<Path>>(
         }
     }
 }
+/// Tries to convert a string slice to a Case
+// TODO convert to RahmenError and remove the unwraps where this is called
+pub fn str_to_case(s: &str) -> Result<Case, String> {
+    // TODO pub fn str_to_case(s: &str) -> RahmenResult<Case> {
+    let case_str = s.to_case(Case::Flat);
+    for case in Case::all_cases() {
+        if case_str == format!("{:?}", case).to_case(Case::Flat) {
+            return Ok(case);
+        }
+    }
+    Err(format!("Unknown Case for conversion: {:}", &s))
+    // TODO Err(RahmenError::CaseUnknown)
+}
 
 #[derive(Debug)]
 enum StatusLineTransformation {
     RegexReplace(Regex, String),
     Capitalize,
+    ChangeCase(String, String),
```

Comment:
This requires that the case string needs to be converted to a case variant on every tag, which is not ideal. It would be better to store the case variants as part of the `ChangeCase` variant, so the conversion only needs to happen once when creating the variant.

Reply:
do you mean this repeat fn call?
`for case in Case::all_cases() {... `
 (sorry, don't know how to link to code here)

should we assign this to a variable beforehand?